.github/workflows/windows-build.yaml

```diff
@@ -89,16 +83,24 @@ jobs:
       with:
         go-version: 1.14
     - name: build runner
-      run: go build -o suricatarunner.exe suricatarunner-windows.go
+      run: |
+        go build -o suricatarunner.exe go/runner/main.go
+        go build -o suricataupdater.exe go/updater/main.go
     - name: add brim files
       run: |
         cp brim-conf.yaml /home/runneradmin/suricata
-        cp suricatarunner.exe /home/runneradmin/suricata/suricatarunner.exe
+        cp suricatarunner.exe suricataupdater.exe /home/runneradmin/suricata/
+    - name: freeze suricata-update
+      run: |
+        curl -L https://github.com/brimsec/suricata-update/archive/fix-windows.tar.gz | tar zxvf - --strip-components=1
+        pip3 install pyinstaller
+        pyinstaller --onefile bin\suricata-update
+        cp dist\suricata-update.exe C:\msys64\home\runneradmin\suricata\bin
+      shell: cmd
     - name: create zip
       run: |
-        cd /home/runneradmin && zip -r suricata-v5.0.3-brim10.$(go env GOOS)-$(go env GOARCH).zip suricata
-    - if: github.ref == 'refs/heads/master'
+        cd /home/runneradmin && zip -r suricata-v5.0.3-brim11.$(go env GOOS)-$(go env GOARCH).zip suricata
+    - if:
```

Comment:
I still want to make one more check of the windows build, but will revert this line before merging.